src/utils/compose.js

```diff
@@ -7,7 +7,7 @@ import omit from 'lodash/object/omit';
 import {
   parseDesc,
   wrapMethods,
-} from '.';
+} from './';
```

Comment:
For some strange reason the merge of PR #16 was never actually merged... unless I am missing something.